### ISSUE TITLE
Crashes in landscape mode

### ISSUE THREAD

**Issue by 06Games** [ConorTheDev/LockWidgets#3]
Hello,
Your tweak is very useful to me, but it crashes when I want to open the notification center in landscape mode. The phone respring  and immediately goes into safe mode.

I use an iPhone 6s under iOS 12.4, jailbroken with Chimera.
The error is always:
```
Exception type: NSInvalidArgumentException
Reason: -[WGWidgetPlatterView setSizeToMimic:]: unrecognized selector sent to instance SOMETHING
```
I have attached a complete [log file](https://github.com/ConorTheDev/LockWidgets/files/3933231/crash.log).

**Comment by conorthedev** [comment]
Hey, I’ll work on this tonight. Thanks for the log! It seems to only occur on iOS 12.* so it’ll be hard to test but I’ll do something :p

**Comment by conorthedev** [comment]
We have a theoretical fix for this issue that I am implementing as we speak, would you be willing to test? @06Games

**Comment by 06Games** [comment]
Yes of course, how can I download it?

**Comment by conorthedev** [comment]
Join my discord: https://oh-my-god.wtf/conor and I’ll send it to you

**Comment by conorthedev** [comment]
This was fixed in 87ac5b5c34d7e3a27fe9f0a5aee51c0c421d41a3, closing
